suite-common/suite-types/src/messageSystem.ts

```diff
@@ -8,7 +8,7 @@
  */
 export type DateTime = string;
 export type Version = string | string[];
-export type Model = '1' | 'T' | 'T1B1' | 'T2T1' | 'T2B1' | 'Safe 3' | '';
+export type Model = '1' | 'T' | 'T1B1' | 'T2T1' | 'T2B1' | 'Safe 3' | 'T3T1' | '';
```

Comment:
Where is this from?

Reply:
generated from `config.schema.v1.json`, resolved on develop....